### PR TITLE
Email template

### PR DESCRIPTION
**Pull request by ccaylak**
I haven't implemented tests yet.

### CODE REVIEW

File: src/main/java/de/adesso/kicker/email/EmailService.java [hunk 1]
```diff
@@ -5,35 +5,38 @@
...
+            MimeMessageHelper messageHelper = new MimeMessageHelper(mimeMessage);
+            messageHelper.setFrom(match.getTeamAPlayer1().getEmail());
+            messageHelper.setTo(matchVerificationRequest.getReceiver().getEmail());
+            messageHelper.setSubject(setSubject(match));
+            messageHelper.setText(verificationText(matchVerificationSentEvent), true);
+        };
+        mailSender.send(messagePreparator);
     }
 
     private String setSubject(Match match) {
```
Comment (Flo4l) [inline src/main/java/de/adesso/kicker/email/EmailService.java]: The subject does not use a text from the properties

File: src/main/java/de/adesso/kicker/email/EmailService.java [hunk 2]
```diff
@@ -5,39 +5,44 @@
...
 
 @Service
 @RequiredArgsConstructor
 public class EmailService {
 
+    private final JavaMailSender mailSender;
+
+    private final EmailMessageBuilder emailMessageBuilder;
+
     static final String ACCEPT_URL = "http://localhost/notifications/accept/";
```
Comment (czarnecki) [inline src/main/java/de/adesso/kicker/email/EmailService.java]: The URL's can be part of the template

File: src/main/java/de/adesso/kicker/email/EmailService.java [hunk 3]
```diff
@@ -5,39 +5,44 @@
...
         var match = matchVerificationRequest.getMatch();
-
-        var simpleMailMessage = new SimpleMailMessage();
-        simpleMailMessage.setFrom(match.getTeamAPlayer1().getEmail());
-        simpleMailMessage.setTo(matchVerificationRequest.getReceiver().getEmail());
-        simpleMailMessage.setSubject(setSubject(match));
-        simpleMailMessage.setText(verificationText(matchVerificationSentEvent));
-
-        mailSender.send(simpleMailMessage);
+        MimeMessagePreparator messagePreparator = mimeMessage -> {
```
Comment (czarnecki) [inline src/main/java/de/adesso/kicker/email/EmailService.java]: I think putting this into it's own function would make the code easier to read

**Comment by ccaylak** [comment]
> I also think renaming the class to `SendVerificationMailService` or someting along those line might be a good idea. Also tests are missing.

>I haven't implemented tests yet.

**Comment by czarnecki** [comment]
> I haven't implemented tests yet.

I read that and this won't be merged unless there are tests.

File: src/main/java/de/adesso/kicker/email/EmailService.java (hunk 2 above)
Comment (ccaylak) [reply src/main/java/de/adesso/kicker/email/EmailService.java]: > > I haven't implemented tests yet.
> 
> I read that and this won't be merged unless there are tests.

I thought that would be obvious, that it's not getting merged without tests

**Pull request merged**